src/app/profile/my-spaces/my-spaces.component.ts

```diff
@@ -156,6 +156,7 @@ export class MySpacesComponent implements OnDestroy, OnInit {
 
   filterChange($event: FilterEvent): void {
     this.applyFilters($event.appliedFilters);
+    this.sort();
```

Comment:
The fix is fine. The code in the file is a bit strange as it also has:

```
  updateSpaces(): void {
    this.applyFilters(this.appliedFilters);
    this.sort();
  }
```

I think there's an opportunity to clean it up a little bit.